developersociety/django-maskpostgresdata
# Add methods for use when subclassing command

Fixes #5. And I think fixes #6 (unless there are other defaults I'm missing). Also fixes #13 by hashing the password used in the default password method.

To test:

* Clone the repo and check out this branch
* `pip install -e ../django-maskpostgresdata` in another of our projects
* Create a management command subclassing this management command (`from maskpostgresdata import BasePostgresDataMaskingCommand`)
* Create a method on this command using the format `update_{db_table_name_of_model}`
* Inside this method specify the function you want to use to update the queryset that gets passed into your method. Copy this one for guidance:

```
    def update_auth_user(self, queryset):
        return queryset.update(password=make_password("password"))
```

The README for this project will be updated by my next PR. So it's no longer 100% accurate based on these changes.
